web-test-runner.config.ts

```diff
@@ -108,20 +110,22 @@ const testRunnerHtml = (
 ): string => `
 <!DOCTYPE html>
 <html lang='en'>
-  <head>${['Roman', 'Bold', 'Light']
-    .map(
-      (type) => `
-    <link
-      rel="preload"
-      href="https://cdn.app.sbb.ch/fonts/v1_6_subset/SBBWeb-${type}.woff2"
-      as="font"
-      type="font/woff2"
-      crossorigin="anonymous"
-    />`,
-    )
-    .join('')}
+  <head>
     <link rel="modulepreload" href="/src/elements/core/testing/test-setup.ts" />
     <style type="text/css">${renderStyles()}</style>
+    <style type="text/css">
+      ${preloadedFonts
+        .map(
+          (f) => `
+      @font-face {
```

Comment:
In the standard theme we already have font face declarations. Are you sure that those ones don't get loaded beside the ones which are preloaded now?

Reply:
In theory, the preloaded rule should win since it comes after the standard theme.
Anyway, it's worth a test just to be sure

Reply:
Already verified, it only loads the right one :)